src/parameterizations/lateral/MOM_mixed_layer_restrat.F90

```diff
@@ -807,6 +808,7 @@ subroutine mixedlayer_restrat_Bodner(CS, G, GV, US, h, uhtr, vhtr, tv, forces, d
     wpup                  ! Turbulent vertical momentum [L H T-2 ~> m2 s-2 or kg m-1 s-2]
   real :: uDml_diag(SZIB_(G),SZJ_(G))  ! A 2D copy of uDml for diagnostics [H L2 T-1 ~> m3 s-1 or kg s-1]
   real :: vDml_diag(SZI_(G),SZJB_(G))  ! A 2D copy of vDml for diagnostics [H L2 T-1 ~> m3 s-1 or kg s-1]
+  real :: lf_bodner_diag(SZI_(G),SZJ_(G)) ! Front width as in Bodner et al., 2023 (B22), eq 24 [L ~> m]
```

Comment:
As the code is currently written, the units of lf_bodner diag should be [Z2 H-1 ~> m or m4 kg-1].

Reply:
I've restored `lf_bodner` back to `L` so this should no longer be necessary.

Reply:
I've restored `lf_bodner` back to `L` so this should no longer be necessary.